### DISCUSSION TITLE
Crystal Maiden 25lvl right talent: Frostbite roots through spell immunity

### DISCUSSION THREAD

**Discussion by Philosoph228** [ValveSoftware/Dota2-Gameplay#2393] [Ideas]
As the enemies gather 5 BKBs, this renders out Crystal Maiden as an useless hero in late game. There no control nor damage through spell immunity.
It would be great if there was a such upgrade for CM to able to, at least, control enemy heroes in BKB, even without causing magical damage from her spells.

**Comment by mrstealyommr** [comment]
Wait out the bkbs and jump in. Ive had cm carry a game like that. Everyone dive bkbs pop off. some kiting. Bkb runs out enemy gets a kill your team gets a kill then cm comes in ults and gets 4 kills. Its fine as it is just play strategically instead of brute forcing it with 25 talents lol.
